src/LocalDelivery/Controller/DeliveryBoyStorefrontController.php

```diff
@@ -0,0 +1,165 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shopware\Production\LocalDelivery\Controller;
+
+use Shopware\Core\Framework\Routing\Annotation\RouteScope;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Production\LocalDelivery\Services\DeliverBoyLoginService;
+use Shopware\Production\LocalDelivery\Services\DeliveryBoyRegisterService;
+use Shopware\Storefront\Controller\StorefrontController;
+use Shopware\Storefront\Framework\Routing\Router;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Twig\Environment;
+
+/**
+ * @RouteScope(scopes={"storefront"})
+ */
+class DeliveryBoyStorefrontController extends StorefrontController
+{
+    /**
+     * @var Environment
+     */
+    private $twig;
+
+    /**
+     * @var DeliveryBoyRegisterService
+     */
+    private $deliveryBoyRegisterService;
+
+    /**
+     * @var DeliverBoyLoginService
+     */
+    private $deliverBoyLoginService;
+
+    /**
+     * @var Router
+     */
+    private $router;
+
+    public function __construct(
+        Environment $twig,
+        DeliveryBoyRegisterService $deliveryBoyRegisterService,
+        DeliverBoyLoginService $deliverBoyLoginService,
+        Router $router
+    ) {
+        $this->twig = $twig;
+        $this->deliveryBoyRegisterService = $deliveryBoyRegisterService;
+        $this->deliverBoyLoginService = $deliverBoyLoginService;
+        $this->router = $router;
+    }
+
+    /**
+     * @Route("/deliveryboy/form", name="delivery.boy.form", methods={"GET"})
+     */
+    public function deliveryBoyRegistrationForm(): Response
+    {
+        $html = $this->twig->render('@LocalDelivery/page/form.html.twig');
```

Comment:
Please use $this->renderStorefront. 
It does some magic on seo urls. Otherwise some links will be broken